reference/test/matrix/csr_kernels.cpp

```diff
@@ -223,6 +249,14 @@ TEST_F(Csr, ConvertsToCoo)
 }
 
 
+TEST_F(Csr, ConvertsToSellp)
+{
+    auto sellp_mtx = gko::matrix::Sellp<>::create(mtx->get_executor());
+    mtx->convert_to(sellp_mtx.get());
+    assert_equal_to_mtx(sellp_mtx.get());
+}
```

Comment:
You also need to test:
- `move_to` sellp (to test the `core` function)
- `calculate_total_cols` kernel